CONTRIBUTING.md

```diff
@@ -5,8 +5,8 @@
 
 ### Prerequisites
 
-1. [Node.js](https://nodejs.org/) 14.16
-1. npm 6.13.4
+1. [Node.js](https://nodejs.org/) 16.14.2
+1. npm 8.13.0
```

Comment:
Looks like 8.13.0 was used when generating the pakcage lock in the past, that seems to be the best match.
8.3.0 makes a tonne of changes, and 8.13.0 has the least changes

Reply:
Still need to test the build and the vsix